### DISCUSSION TITLE
Renovate Dependency Manager

### DISCUSSION THREAD

**Discussion by alanlujan91** [econ-ark/HARK#1683] [General]
We've had a number of discussions and frustrations with managing dependencies. 

I came across [renovatebot](https://www.mend.io/renovate/) which helps keep dependencies updated automatically. 

There's also different dependency managers which we could take a look at.

**Comment by llorracc** [comment]
One of the things on @Cameron Riddell ***@***.***> 's agenda is to
help us rationalize and systematize dependency management for REMARKs, so
I'm forwarding this to him.



On Thu, Mar 23, 2023 at 1:58 PM Alan Lujan ***@***.***> wrote:

> We've had a number of discussions and frustrations with managing
> dependencies.
>
> I came across renovatebot <https://www.mend.io/renovate/> which helps
> keep dependencies updated automatically.
>
> There's also different dependency managers which we could take a look at.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/econ-ark/HARK/issues/1247>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAKCK72FV56DLGI2LNVDPLDW5SFLTANCNFSM6AAAAAAWFRN34M>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
- Chris Carroll

**Comment by mnwhite** [comment]
Is this part of the "cookie" PR now?
